backend/src/main/java/com/salat/briene/controlleradvices/UserAdvice.java

```diff
@@ -0,0 +1,32 @@
+package com.salat.briene.controlleradvices;
+
+import com.salat.briene.exceptions.AnonymousUserException;
+import com.salat.briene.exceptions.DuplicatedUserException;
+import com.salat.briene.exceptions.UserNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class UserAdvice {
+    @ExceptionHandler(AnonymousUserException.class)
+    public ResponseEntity<?> handleAnonymousUser(AnonymousUserException e) {
+        e.printStackTrace();
+        return ResponseEntity.status(HttpStatus.UNAUTHORIZED).body(e.getMessage());
+    }
+
+    @ExceptionHandler(DuplicatedUserException.class)
+    public ResponseEntity<?> handleDuplicatedUser(DuplicatedUserException e) {
+        e.printStackTrace();
```

Comment:
1) We just print this exception to console (without timestamp, thread, class,) - it's not the right way for logging. We always need to use loggers (slf4j for example)
2) Do we need to log this errors? It's not application, but user-side error. We can log them, but on debug level in my opinion.